scripts/src/main/resources/scripts/command/lazydocker

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env bash
+
+# autocompletion list
+if [ "${1}" = "shortlist" ]
+then
+  if [ -z "${2}" ]
+  then
+    echo "setup help"
+  fi
+  exit
+fi
+
+if [ -n "${DEVON_IDE_TRACE}" ]; then set -vx; fi
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+
+# Call lazydocker with specified arguments.
+function doRun() {
+  doSetup silent
+  cd "${LAZYDOCKER_HOME}" || exit 1
+  if doIsWindows
+  then
+    powershell.exe -command 'start powershell {.\lazydocker.exe}'
+  else
+    ./lazydocker 
+  fi
+}
+
+function doSetup() {
+  if [ ! -d "${LAZYDOCKER_HOME}" ] || [ "${1}" != "silent" ]
+  then
+    local version="${LAZYDOCKER_VERSION}"
+    # Get latest release
+    if [ -z "${version}" ]
+    then 
+      version="0.18.1"
+      doEcho "Getting latest release..."
+      local latestversion
+      latestversion=$(curl -Ls -o /dev/null -w '%{url_effective}' https://github.com/jesseduffield/lazydocker/releases/latest | sed 's/^.*\///' | sed 's/^v//')
+      doVersionCompare "${latestversion}" "${version}"
+      if [ "${?}" = 1 ]
+      then
+        version=${latestversion}
+      fi
+    fi
+    doCheckDockerVersions
+    doInstall "-" "${LAZYDOCKER_HOME}" "lazydocker" "${version}"
+  fi
+}
+
+function doCheckDockerVersions() {
+  local DOCKERENGVERSION
+  DOCKERENGVERSION=$(docker version | grep "Version" | head -1 | sed -ne 's/.*\([0-9][0-9]\+\.[0-9]\+\.[0-9]\+\).*/\1/p')
+  local DOCKERAPIVERSION
+  DOCKERAPIVERSION=$(docker version | grep "API version" | head -1 | sed -ne 's/.*\([0-9]\+\.[0-9]\+\).*/\1/p')
+  doVersionCompare "1.3.0" "${DOCKERENGVERSION}"
+  local res1=$?
+  doVersionCompare "1.25" "${DOCKERAPIVERSION}"
+  local res2=$?
+  local ressum=$((res1 + res2))
+  doWarning "Please be aware, that if you want to use docker-compose with lazydocker, then you'll need at least version 1.23.2 of docker-compose"
+  if [ "${res1}" == "1" ] || [ "${res2}" == "1" ]
+  then
+    if [ "${res1}" == "1" ]
+    then
+      doWarning "\nIt seems as if your docker-engine version is too old for a proper usage of lazydocker.\nPlease upgrade it at least to version 1.3.0"
+      read -r -p "Continue by pressing enter"
+    fi
+    if [ "${res2}" == "1" ]
+    then
+      doWarning "\nIt seems as if your docker-API version is too old for a proper usage of lazydocker.\nPlease upgrade it at least to version 1.25.0"
+      read -r -p "Continue by pressing enter"
+    fi
+  elif [ "${ressum}" == "4" ] || [ "${ressum}" == "0" ] || [ "${ressum}" == "2" ]
```

Comment:
so `4`  means that both versions are higher and `0` that both versions are identical. So `2` means one is equal and one is higher - because you are in the else so it can not be the `1+1` case? Somehow seems correct but does this `ressum` really make anything better? IMHO it is only causing confusion and making the code hard to read.

Why not just kick out `ressum` and do `else` here?
The only possible results are 0 (equal), 1 (greater), 2 (lower):
https://github.com/devonfw/ide/blob/3c49143d7a3167378f90a25daf09b1d890d2dd27/scripts/src/main/resources/scripts/functions#L1381

To avoid confusion it would even be helpful to switch the arguments:
```
doVersionCompare "${DOCKERENGVERSION}" "1.3.0"
local res1=$?
```

so `1` means greater (${DOCKERENGVERSION} > 1.3.0) and `2` means lower (${DOCKERENGVERSION} < 1.3.0).

So these are the cases (without switching version arguments):
|res1|res2|result|
|----|----|------|
|  0  | 0  | fine, both versions equal |
|  0 |  1  | bad, ApiVersion is lower |
|  1 |  0  | bad, Version is lower |
|  1 |  1  | bad, both versions lower |
|  0 |  2  | fine, version equal, ApiVersion higher |
|  1 |  2  | bad, version lower, ApiVersion higher |
|  ... |  ... | ... |

So to make it short, the first if is covering all bad cases, and `else` would be fine.